queries.sql

```diff
@@ -8,3 +8,45 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE (weight_kg >= 10.4) AND (weight_kg <= 17.3);
+
+/* Transaction */
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
```

Comment:
- [ ] Good job so far, but as specified in the project requirements please make sure to verify the changes before and after rolling back at line `16`.